src/platform/lumin-runtime/mxs-base-app.js

```diff
@@ -82,15 +82,18 @@ export class MxsBaseApp {
     return prismController.getContainer(nodeName);
   }
 
-
   addPrism(properties, app) {
     const prismSize = properties.size;
-
     if (!Array.isArray(prismSize)) {
       throw new TypeError(`Prism size is not a vec3: ${prismSize}`);
     }
 
-    const prism = executor.callNativeFunction(app, 'requestNewPrism', prismSize);
+    const prismType = properties.type !== undefined ? properties.type : 'world';
```

Comment:
We should call out in the documentation that `World` is the default.

Reply:
Agree ! The docs need to be updated.